rules/Php80/NodeAnalyzer/PhpAttributeAnalyzer.php

```diff
@@ -32,6 +37,31 @@ public function hasPhpAttribute(Property | ClassLike | ClassMethod | Param $node
         return false;
     }
 
+    public function hasInheritedPhpAttribute(ClassLike $classLike, string $attributeClass): bool
+    {
+        $className = $classLike->namespacedName->toString();
+        $reflectionClass = $this->reflectionProvider->getClass($className);
```

Comment:
should have check classname exists first:

```suggestion
        if (! $this->reflectionProvider->getClass($className)) {
            return false;
        }

        $reflectionClass = $this->reflectionProvider->getClass($className);
```

Reply:
This change seems to result in PHPstan complaining that it always evaluates to false.

Reply:
oh yes, should be use `hasClass()`

Reply:
The change should be:

```suggestion
        if (! $this->reflectionProvider->hasClass($className)) {
            return false;
        }

        $reflectionClass = $this->reflectionProvider->getClass($className);
```